demo/Signs.swift

```diff
@@ -1067,6 +1175,767 @@ extension Sign {
+                } else {
+                    return nil
+                }
+            case .warningRoundabout:
+                if country == .UK {
+                    return Asset.Signs.warningRoundAboutUK
+                } else if country == .other {
+                    return Asset.Signs.warningRoundAboutEU
+                } else {
+                    return nil
+                }
+            case .warningSchoolZone:
+                if country == .UK {
+                    return Asset.Signs.warningSchoolZoneUK
+                } else if country == .other {
+                    return Asset.Signs.warningSchoolZoneEU
+                } else {
+                    return nil
+                }
+            case .warningSlipperyRoadSurface:
+                return Asset.Signs.warningSlipperyRoadSurfaceEU
+            case .warningSoftShoulderLeft:
+                return Asset.Signs.warningSoftShoulderLeftEU
+            case .warningSoftShoulderRight:
+                return Asset.Signs.warningSoftShoulderRightEU
+            case .warningSpeedBump:
+                return Asset.Signs.warningSpeedBumpEU
+            case .warningSteepAscent:
+                return Asset.Signs.warningSteepAscentEU
+            case .warningSteepDescent:
+                return Asset.Signs.warningSteepDescentEU
+            case .warningTrafficQueues:
+                if country == .UK {
+                    return Asset.Signs.warningTrafficQueuesUK
+                } else if country == .other {
+                    return Asset.Signs.warningTrafficQueuesEU
+                } else {
+                    return nil
+                }
+            case .warningTrafficSignals:
+                return Asset.Signs.warningTrafficSignalsEU
+            case .warningTramsCrossing:
+                return Asset.Signs.warningTramsCrossingEU
+            case .warningTwoWayTraffic:
+                if country == .UK {
+                    return Asset.Signs.warningTwoWayTrafficUK
+                } else if country == .other {
+                    return Asset.Signs.warningTwoWayTrafficEU
+                } else {
+                    return nil
+                }
+            case .warningUnevenRoad:
+                return Asset.Signs.warningUnevenRoadEU
+            case .warningWildAnimals:
+                if country == .UK {
+                    return Asset.Signs.warningWildAnimalsUK
+                } else if country == .other {
+                    return Asset.Signs.warningWildAnimalsEU
+                } else {
+                    return nil
+                }
+            case .warningWindingRoad:
+                if country == .UK {
+                    return Asset.Signs.warningWindingRoadUK
+                } else if country == .other {
+                    return Asset.Signs.warningWindingRoadEU
+                } else {
+                    return nil
+                }
+            case .speedLimitNight:
```

Comment:
I think we should use here `default` too

Reply:
Same as above. What do you think?

Reply:
Update: squash nil-cases under common `default`.